lib/index.js

```diff
@@ -568,6 +568,8 @@ function prependLibraryPath(path) {
 // Public API
 exports.require = giRequire
 exports.startLoop = internal.StartLoop
+exports.setProperty = internal.ObjectPropertySetter
+exports.getProperty = internal.ObjectPropertyGetter
```

Comment:
Public API requires documentation. For the names, I would prefer `(s|g)etObjectProperty`.

Reply:
Will add this into a different issue/PR